packages/seacas/libraries/ioss/src/Ioss_DatabaseIO.h

```diff
@@ -279,6 +279,18 @@ namespace Ioss {
       return create_subgroup_nl(group_name);
     }
 
+    bool open_root_group()
```

Comment:
I initially thought that this would be case when I originally implemented it that way. The problem is that if I am currently in a subgroup and I call `open_group("/")`, the file pointer that is returned from 
`ex_get_group_id(get_file_pointer(), "/", &m_exodusFilePtr);`
is exactly the file pointer returned from `get_file_pointer()` so that simply returns the same subgroup instead of the root group

Reply:
OK.  I looked at the exodus function and it says that the root group is returned if name is passed in as NULL and not "/" although that should fall down to the full_ncid path and return the root also...

I've modified `ex_get_group_id` to check whether filename is NULL or "/" and should now return the correct root id.  Will commit this fix soon.

Reply:
Ok, but are you still good with leaving this as a convenience function ?

Reply:
Yes, but maybe it just calls through to the other function... Or has much less in the guts.

Reply:
Ok, I'll wait for your commit to go through